### PR TITLE
javascript/javascript1/week3

### PR DESCRIPTION
**Pull request by arya56**

### CODE REVIEW

File: javascript/javascript1/week3/speed.js [hunk 1]
```diff
@@ -0,0 +1,12 @@
+const travelInformation = {
+    speed: 50,
+    destinationDistance: 432,
+  };
+  function notThisFunctionName(travelInformation){
```
Comment (dariye) [inline javascript/javascript1/week3/speed.js:5]: You should rename this function to something more meaningful. One helpful way is to think in terms of what the function is doing.

File: javascript/javascript1/week3/speed.js [hunk 2]
```diff
@@ -0,0 +1,12 @@
...
+      const hoursOf =Math.trunc(travelInformation.destinationDistance / travelInformation.speed);
```
Comment (dariye) [inline javascript/javascript1/week3/speed.js:6]: Out of curiosity, why are you using the method `Math.trunc`?

File: javascript/javascript1/week3/smart-ease.js [hunk 3]
```diff
@@ -0,0 +1,149 @@
...
+}
+
+saveNote("Pick up groceries", 1);
+saveNote("Do laundry", 2);
+saveNote("Wash the dishes", 3);
+saveNote("Take a shower", 4);
+
+function getNote(id) {
+  for (let i = 0; i < notes.length; i++) {
+    if (!id || isNaN(id)) {
```
Comment (dariye) [inline javascript/javascript1/week3/smart-ease.js:13]: Out of curiosity, why are you using `.isNaN`?

File: javascript/javascript1/week3/smart-ease.js [hunk 4]
```diff
@@ -0,0 +1,149 @@
...
+      `The note with id: ${notes[i].id}, has the following note text: ${notes[i].content}`
+    );
+  }
+}
+
+
+//   addActivity
+const activities = [];
+function addActivity(date, activity, duration) {
+  if (typeof date === "string" && typeof activity === "string" && !isNaN(duration)) {
```
Comment (dariye) [inline javascript/javascript1/week3/smart-ease.js:38]: Neat! Great way to enforce the right data structure for activity

File: javascript/javascript1/week3/smart-ease.js [hunk 5]
```diff
@@ -0,0 +1,149 @@
+const notes = [];
+function saveNote(content, id) {
+  notes.push({ content: content, id: id });
```
Comment (dariye) [inline javascript/javascript1/week3/smart-ease.js:3]: There's a nice little js feature that lets you do this
```suggestion
  notes.push({ content, id });
```

JavaScript will handle the assignment for you so it's equivalent to what you have
```
notes.push({ content: content, id: id });

```

You only do this when the property has the same name as the variable you're assigning it to.

File: javascript/javascript1/week3/speed.js (hunk 2 above)
Comment (arya56) [reply javascript/javascript1/week3/speed.js:6]: I changed it with Math.floor!

File: javascript/javascript1/week3/smart-ease.js (hunk 3 above)
Comment (arya56) [reply javascript/javascript1/week3/smart-ease.js:13]: In explanation of this exercise it says "If no id is specified or if the id is not a number, log out an error string."

**Pull request merged**